src/current/v24.3/sso-sql.md

```diff
@@ -77,13 +79,13 @@ You can also view all of your cluster settings in the DB Console.
     This must match your cluster's configured value for `server.oidc_authentication.provider_url`. Refer to [Single Sign-on (SSO) for DB Console]({% link {{ page.version.version }}/sso-db-console.md %}#cluster-settings). Issuers are expected to publish their configuration at `https://{ domain }/.well-known/openid-configuration`. For example:
 
         - CockroachDB {{ site.data.products.cloud }}'s IdP configuration can be viewed publicly at: `https://cockroachlabs.cloud/.well-known/openid-configuration`.
-        The `issuer` is `https://cockroachlabs.cloud`.
+        The issuer URI is `https://cockroachlabs.cloud`.
```

Comment:
The convention that we've been following is to use URL for issuer, and URI for JWKS.